app_test.go

```diff
@@ -100,6 +100,57 @@ func TestInvokes(t *testing.T) {
 	})
 }
 
+func TestError(t *testing.T) {
+	t.Run("NilErrorOption", func(t *testing.T) {
+		var invoked bool
+
+		app := fxtest.New(t,
+			Error(nil),
+			Invoke(func() { invoked = true }), // should not be called
```

Comment:
nit: instead of tracking `invoked` like this, you can mark the test as failed.

```
Invoke(func() { t.Errorf("Invoke should not be called") }),
```

Reply:
Also, in this case, the comment appears to be incorrect. You *do* expect
Invoke to be called here. Tracking the bool for that case is fine.

Reply:
Good catch, updated.